pkg/auth/tokenUtils.go

```diff
@@ -0,0 +1,53 @@
+package auth
```

Comment:
AFAIK, [file names should be all lowercase](https://stackoverflow.com/a/39625565) (same for other occurrences)